android/app/proguard-rules.pro

```diff
@@ -157,3 +157,11 @@
 -keep class org.spongycastle.**
 -dontwarn org.spongycastle.jce.provider.X509LDAPCertStoreSpi
 -dontwarn org.spongycastle.x509.util.LDAPStoreHelper
+
+##### Glide #####
+-keep public class * implements com.bumptech.glide.module.GlideModule
```

Comment:
Was it necessary to add these instructions?

Reply:
It turned out that these are not needed 😁